force-app/main/default/objectTranslations/Application__c-ca/Application_Status__c.fieldTranslation-meta.xml

```diff
@@ -47,4 +47,8 @@
         <masterLabel>Declined Offer</masterLabel>
         <translation>Oferta rebutjada</translation>
     </picklistValues>
+    <picklistValues>
```

Comment:
The new "Started" value is added at the end of picklist values. I am wondering if it should be added before "Submitted"? @MatthewBlanski , @vmarasow 
<img width="785" alt="Screen Shot 2021-07-19 at 2 34 27 PM" src="https://user-images.githubusercontent.com/28743286/126230620-b7e7a27d-fa1c-434f-a709-e9895a59af77.png">
<img width="823" alt="Screen Shot 2021-07-19 at 2 36 16 PM" src="https://user-images.githubusercontent.com/28743286/126230694-e6e93cc0-2cae-444f-b09e-816fcea26953.png">

Reply:
@JasonFitzy , can you please check into Application Status ordering with regards to where new value "Started" should be added? Thank you.

Reply:
@RadSFDeveloper Thanks for asking! EDA picklists are either in alphabetical order or in an order due to some "flow" the data speaks to. As you astutely pointed out when thinking about the flow of application status "Started" should be before "Submitted". I agree. Can we put it in that position? 

and sorry on the delay I normally do not get questions in GitHub. :)

Reply:
It is all good, there is no delay, thanks for the response @JasonFitzy !